main/communication/index.ts

```diff
@@ -50,28 +17,40 @@ export default class Communication {
   private bindEvents(): void {
     // Bind events to props
     [
+      { event: 'electraJs:wallet:addresses', instance: this.electraJs.wallet, prop: 'addresses' },
+      { event: 'electraJs:wallet:allAddresses', instance: this.electraJs.wallet, prop: 'allAddresses' },
+      { event: 'electraJs:wallet:daemonState', instance: this.electraJs.wallet, prop: 'daemonState' },
       { event: 'electraJs:wallet:isNew', instance: this.electraJs.wallet, prop: 'isNew' },
       { event: 'electraJs:wallet:lockState', instance: this.electraJs.wallet, prop: 'lockState' },
+      { event: 'electraJs:wallet:masterNodeAddress', instance: this.electraJs.wallet, prop: 'masterNodeAddress' },
```

Comment:
What is this masterNodeAddress? It's not being used for anything right?

Reply:
It is used in `saveUserSettings()` (in Login component).